src/Jmikola/AutoLogin/Http/Firewall/AutoLoginListener.php

```diff
@@ -34,11 +35,11 @@ class AutoLoginListener implements ListenerInterface
      * @param AuthenticationManagerInterface                 $authenticationManager
      * @param string                                         $providerKey
      * @param string                                         $tokenParam
-     * @param LoggerInterface                                $logger
+     * @param PsrLoggerInterface|HttpKernelLoggerInterface   $logger
```

Comment:
Sounds like a good idea. The PSR-3 logger has actually been extended since v2.2.0: https://github.com/symfony/symfony/commit/91a86f8bec0e1653cc976ac3fc6163b43c40acfa, so it's even less of a concern.